.env.sample

```diff
@@ -0,0 +1,4 @@
+FLASK_DEBUG=True
+CLIENT_ID=<client id>
+CLIENT_SECRET=<client secret>
+TENANT_ID=<tenant id>
```

Comment:
In my PR on the docs repo, I actually remove the Portal option entirely. Given we haven't heard back from the Portal team yet, it feels like it's not a very maintainable path. It's also not that hard to do the manual configuration - it's even gotten a bit easier since they first wrote the article.

But I don't know if everyone's on board with that, I haven't heard from the reviewer. What do you think?

Reply:
I like your [removal of Portal option in the docs repo](https://github.com/MicrosoftDocs/azure-docs-pr/pull/227776/files#r1115181458).

But, even after your removal, the quickstart functionality in Portal is still anticipating those "Enter_the_Application_Id_here", "Enter_the_Client_Secret_here" macro names to exist in this sample. That is why I thought it might be better if we keep those "Enter_the_Application_Id_here" instead of the new `<client id>`. That being said, last time you also showed me that the Portal's quickstart seems to generate a new sample.zip on-the-fly, presumably based on an old snapshot that still has those "Enter_the_Application_Id_here" inside. So, perhaps we are free to change them to `<client id>` here. I am fine with your current change here, then.

Reply:
I have updated the Powershell scripts that referenced "Enter_the_Application_Id_here" and app_config.py, but still need to go through the B2C readme's references. I think that's the only pending TODO for this PR.